cmd/oci-create-runtime-bundle/main.go

```diff
@@ -71,7 +71,7 @@ func newBundleCmd(stdout, stderr *log.Logger) *cobra.Command {
 	cmd.Flags().StringVar(
 		&v.typ, "type", "",
 		fmt.Sprintf(
-			`Type of the file to unpack. If unset, oci-image-tool will try to auto-detect the type. One of "%s"`,
+			`Type of the file to be unpacked. If unset, oci-create-runtime-bundle will try to auto-detect the type. One of "%s"`,
```

Comment:
I don't think we need the “unpack” → “be unpacked”.

The `oci-image-tool` → `oci-*` changes look good.